include/bluetooth/direction.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_DF_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_DF_H_
+
+/** @brief Constant Tone Extension parameters for connectionless
+ * transmission.
+ *
+ * The structure holds information required to setup CTE transmission
+ * in periodic advertising.
+ */
+struct bt_le_df_adv_cte_tx_params {
+	/* Length of CTE in 8us units */
```

Comment:
Need to use `/**` for doxygen. (rest of this struct)

Reply:
Corrected.